src/sentry/static/sentry/app/components/events/interfaces/spans/spanBar.tsx

```diff
@@ -970,16 +972,16 @@ const getTogglerTheme = ({
 
   if (disabled) {
     return `
-    background: ${buttonTheme.background};
```

Comment:
This feels like a similar situation as border, but it's probably not used as much so this might be ok

Reply:
👍